### ISSUE TITLE
Elite M1 crashes upon closing the WiFi dialog box

### ISSUE THREAD

**Issue by LukeFernandes** [tehmonolith/EliteM1Release#1]
I proceed according the instructions. After connecting my computer to the camera's wifi, I then press 'close' on the wifi dialog box. The Elite M1 screen turns white and then the application crashes.

System: Windows 10 x64

**Comment by Qgrade** [comment]
Exactly the same thing happens to me! OS: W10 x64

**Comment by tehmonolith** [comment]
Hey y'all. Sorry about the delay in getting back to you. I've actually sold my Yi M1 and so won't be able to support this project going forward. As such I'll look to open source what I've done so far so that others might build upon it.

**Comment by Qgrade** [comment]
Yes, it would be fun partly because there is no SDK or API for Y1-M1 and the only way is to communicate (on w10 PC) via http and json request. You have started the interest in a PC app with your application. There is certainly interest in building on it. Thank you very much!

**Comment by LukeFernandes** [comment]
Sad to hear you've sold your Yi M1 but thanks for building this!

If you could open source it, I'd be happy to help develop this. @Qgrade are you able to chip in?

**Comment by Qgrade** [comment]
I would like to see how tehmonolith has done this far, if he wants to. We'll see.

**Comment by LukeFernandes** [comment]
@tehmonolith just a gentle reminder to open source this - would be good to play around with the code, I'd love to one day get an app solution for my Yi M1 :)

**Comment by bmx22c** [comment]
Open sourcing this would be so cool !
